commands/create.go

```diff
@@ -539,11 +539,11 @@ func updateUserdataFile(driverOpts *rpcdriver.RPCFlags, machineName, userdataFla
 // writeCloudConfig sets custom install script path to the runcmd directive
 // and passes the script path to commonCloudConfig
 func writeCloudConfig(machineName, encodedData, machineOS string, cf map[interface{}]interface{}, newUserDataFile *os.File) error {
-	path := "/usr/local/custom_script/install.sh"
+	path := "sh /usr/local/custom_script/install.sh"
```

Comment:
nit: path is no longer a good name for this variable.

Reply:
I switched to using `command`